src/platforms/common/session-replay/index.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Set Up Session Replay
+sidebar_order: 4000
+supported:
+  - javascript
+  - javascript.angular
+  - javascript.ember
+  - javascript.gatsby
+  - javascript.nextjs
+  - javascript.react
+  - javascript.remix
+  - javascript.svelte
+  - javascript.vue
+description: "Learn how to enable session replay in your app if it is not already set up."
+---
+
+With [session replay](/product/session-replay/), TODO.
+
+<PlatformSection supported={["javascript", "javascript.vue", "java.spring", "react-native"]} notSupported={["java", "go", "android", "ruby", "python", "dotnet", "dart", "native"]}>
```

Comment:
You have way more context here but just out of curioisity, `java.spring` really supports Replay? Are they using the Browser SDK?

Also, do we need to explicilty add `javascript.vue` or is it automatically included by `javascript`?

Reply:
ah, this is all copy+paste that i missed. none of these are really supported